internal/db/migrations/005_reservations.sql

```diff
@@ -16,6 +16,10 @@ CREATE TABLE aws_reservation_details
   reservation_id BIGINT NOT NULL,
   provider INTEGER NOT NULL DEFAULT provider_type_aws(),
   pubkey_id BIGINT REFERENCES pubkeys(id),
+  source_id INTEGER NOT NULL,
+  instance_type TEXT NOT NULL,
+  amount INTEGER NOT NULL,
```

Comment:
I'm missing the AMI/image_id information we've talked about today! :thinking:
Should we add one or the other? :)

Reply:
yes, you are right. 
I am just wondering which info is supposed to be saved for a reservation and which for a job 
because in order to perform a job I don't need the source id so it is not needed there, but for a reservation do we want to save all relevant info?

Reply:
Yeah, I'd save all relevant information :thinking:
I think we will make use of the source_id in the future for sure, we need to assume role for actually running the instance on AWS right?

Reply:
good point, maybe in order to run ec2.RunInstance I would need to assume role again :D